app/scripts/Xprize-UserInterfacesRetargeting.xml

```diff
@@ -123,5 +119,11 @@
     <to>/joypadDevice/Oculus/rpc</to>
     <protocol>fast_tcp</protocol>
   </connection>
+  
+  <connection>
+    <from>/WeightRetargeting/output:o</from>
+    <to>/iFeelSuit/WearableActuatorsCommand/input:i</to>
+    <protocol>udp</protocol>
```

Comment:
Depending on where we run the module, at the moment we may not have the udp option for the carrier type. Now, as @S-Dafarra  was mentioning before, we cannot have udp for the double-network configuration. In our case, I think the module will run on robot side, so udp is not a reliable solution. We may use fast_tcp, what do you say @RiccardoGrieco ?

Reply:
To be honest, I am not sure anymore about this. I remember that the audio did not work when using ``udp``. Maybe it was an isolated case, or maybe it got fixed in the meantime. In any case, I agree that ``fast_tcp`` is safer

Reply:
Done in fdd66be